HW11/homework_11.py

```diff
@@ -0,0 +1,49 @@
+import time
+from datetime import datetime
+
+
+class ContextManager:
+    def __enter__(self):
+        print("="*10)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print("="*10)
```

Comment:
Your context manager should print an exception if there is one. And it shouldn't raise it, so use `return True` in the end of the method